cluster/gce/config-default.sh

```diff
@@ -205,13 +205,6 @@ elif [[ "${PREEMPTIBLE_NODE}" == "true" ]]; then
     WINDOWS_NON_MASTER_NODE_LABELS="${WINDOWS_NON_MASTER_NODE_LABELS},cloud.google.com/gke-preemptible=true"
 fi
 
-# To avoid running Calico on a node that is not configured appropriately,
-# label each Node so that the DaemonSet can run the Pods only on ready Nodes.
-# Windows nodes do not support Calico.
-if [[ ${NETWORK_POLICY_PROVIDER:-} == "calico" ]]; then
```

Comment:
There is at least one more reference to `NETWORK_POLICY_PROVIDER` in the tree.

Reply:
I think I found all of them